charts/velocity-operator/templates/crds/sqs/services.k8s.aws_adoptedresources.yaml

```diff
@@ -0,0 +1,226 @@
+---
```

Comment:
You probably need to sorround this with `{{- if .Values.tags.cloudResources }}` like all the other CRDs we include here

Reply:
The same for the other files in this dir